template/koa-rest-api/src/framework/validation.ts

```diff
@@ -26,15 +27,33 @@ import type { Context } from 'src/types/koa';
  * Returns:
  *
  * ```json
- * { "/advertiserId": "advertiserId is required in the URL" }
+ * { "/advertiserId": ["advertiserId is required in the URL"] }
```

Comment:
Do we want to introduce an array in the typical case where there is only one error message?

Reply:
Will revert the change since we are handling the union error with the index now.